docs/mmteb/validate_points.py

```diff
@@ -5,6 +5,17 @@
 from jsonlines import Reader
 from pydantic import BaseModel, ConfigDict, Field, ValidationError, conint, constr
 
+commit_exceptions = {
+    "scores_from_old_system",
+    # <100 points: from before max points were enforced
+    "440",
+    "543",
+    "616",
+    "636",
+    # >100 points: from before max points were enforced (reduced to 100 points)
+    "583",
```

Comment:
In agreement with author

Reply:
@imenelydiaker if we are to be rigid here I believe we should also reduce the point of flores. I don't believe it changes the scores of any from the team to less than 10 (still fairly high).